src/Core/Managed/Shared/Extensibility/TelemetryConfiguration.cs

```diff
@@ -22,13 +22,85 @@ public sealed class TelemetryConfiguration : IDisposable
         private static TelemetryConfiguration active;
 
         private readonly SnapshottingList<ITelemetryInitializer> telemetryInitializers = new SnapshottingList<ITelemetryInitializer>();
+        private ITelemetryChannel telemetryChannel = null;
         private TelemetryProcessorChain telemetryProcessorChain;
         private string instrumentationKey = string.Empty;
         private bool disableTelemetry = false;
         private TelemetryProcessorChainBuilder builder;
         private SnapshottingList<IMetricProcessor> metricProcessors = new SnapshottingList<IMetricProcessor>();
 
         /// <summary>
+        /// Indicates if this instance has been disposed of.
+        /// </summary>
```

Comment:
I feel that this particular comment is noise that only makes reading the code harder. Comments quickly become obsolete and misleading. It's better for the code to be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [self-documenting](https://en.wikipedia.org/wiki/Self-documenting_code) first; and doc-commented only as needed. ReSharper and Ghost Doc make creating useless comments too easy and StyleCop actually encourages the bad practice! 😱 Does a doc comment saying a constructor initializes a new instance of a class have any value?